emoji_presentation_scanner_vs.rl

```diff
@@ -109,5 +116,5 @@ scan_emoji_presentation_vs (emoji_text_iter_t p,
   /* Should not be reached. */
   *is_emoji = false;
   *has_vs = false;
-  return pe;
+  return p;
```

Comment:
Why is this line changed?

Reply:
Because return value is non-const, and `pe` is const. The code says `/* Should not be reached */`. Maybe we should also add an assert there.